index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="responsive-style.css">
+  <script src="script.js" defer></script>
+  <script src="mobileMenu.js" defer></script>
+</head>
+<body>
+  <section id="home">
+    <nav>
+      <a href="index.html"><img src="img/logo.png" alt="" class="cclogo"></a>
+      <img src="img/mobile_menu.png" alt="" class="hamburger">
+      <ul class="link-items">
+        <li><a href="about.html">About</a> <img src="img/close_mobile_menu.png" alt=""></li>
+        <li><a href="index.html">Program</a></li>
+        <li><a href="index.html">Join</a></li>
+        <li><a href="index.html">Sponsor</a></li>
+        <li><a href="index.html">News</a></li>
+        <li><a href="index.html">CC Campaign</a></li>
+      </ul>
+    </nav>
+    <article class="welcome">
+      <p class="hello">"Hello! Sharing World"</p>
+      <h1 class="main-heading">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <P class="joyful">A joyful festival where people from over 80 countries who create positive change gather and share their thoughts will be held in Korea in October.</P>
+      <p class="schedule">2022.10.14 (Wed) ~ 17 (Sat)</p>
+      <p class="museum">@National Museum of Korea, Art Center Nabi, etc.</p>
```

Comment:
- [ ] Nice work @dazehinn, I salute your efforts 👍 , however, my eyes are not convinced yet regarding the design, there are some points to adjust and stick to the given original design

- Let's start with this hero section, the project requirement says to match as much as possible the original design, but we agree that the color and font weight of this 1st text is not the same as the one in the given template, right? yes, so please update that.

   *** `HINT`: There is a CSS property that helps put images as the background of a text, it is the one used in the design, do it and you will have the same style @dazehinn 

- Your paragraph text is not sharing the same color and font-weight as the original design, please fix that...
- Please Update the color, alignment, and font-weight as the original design as pointed out on the left.
- Apart from the requirement, I think the text you just use in red is not suitable though, mixing colors in the middle of the page like that is not fine for the UI.
- You are centering the hero section content which should not be so, seeing the given template, it should be aligned on the left and not in the middle, please update that too

<img width="1173" alt="1" src="https://user-images.githubusercontent.com/77843050/211066671-6628fcbc-a417-4fe7-9bed-9544e13c4f46.png">